collection-scripts/gather

```diff
@@ -96,5 +96,8 @@ oc adm inspect --dest-dir must-gather --rotated-pod-logs "${group_resources_text
 # Gather PodNetworkConnectivityCheck
 /usr/bin/gather_podnetworkconnectivitycheck
 
+# Gather CRO resources
```

Comment:
This should say Gather CMA resources
(not CRO)

Reply:
Fixed. I spent too much time playing with CRO 😄